<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisRangeTestDialog</class>
 <widget class="QDialog" name="AxisRangeTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>434</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Axis Range Test</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLabel" name="pitchCheckPositiveStatus">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/icons/32x32/Images/Ellipsis_32x32.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QFrame" name="buttonsFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="retryButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Retry</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLabel" name="testStatus">
     <property name="text">
      <string>Test Running...</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="yawReturnHomeStatus">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/icons/32x32/Images/Ellipsis_32x32.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="rollReturnHomeStatus">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/icons/32x32/Images/Ellipsis_32x32.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="yawCheckNegativeStatus">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/icons/32x32/Images/Ellipsis_32x32.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="pitchReturnHomeStatus">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/icons/32x32/Images/Ellipsis_32x32.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="pitchCheckNegativeStatus">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/icons/32x32/Images/Ellipsis_32x32.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="rollCheckNegativeStatus">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/icons/32x32/Images/Ellipsis_32x32.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QProgressBar" name="rollCheckNegativeProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QProgressBar" name="yawCheckNegativeProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="rollCheckPositiveStatus">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/icons/32x32/Images/Ellipsis_32x32.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QProgressBar" name="rollReturnHomeProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QProgressBar" name="rollCheckPositiveProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="yawCheckPositiveStatus">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/icons/32x32/Images/Ellipsis_32x32.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="pitchCheckPositive_label">
     <property name="text">
      <string>Pitch Check Positive Range:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="rollCheckNegative_label">
     <property name="text">
      <string>Roll Check Negative Range:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QProgressBar" name="pitchCheckPositiveProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="pitchCheckNegative_label">
     <property name="text">
      <string>Pitch Check Negative Range:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="pitchReturnHome_label">
     <property name="text">
      <string>Pitch Return Home:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="rollCheckPositive_label">
     <property name="text">
      <string>Roll Check Positive Range:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="rollReturnHome_label">
     <property name="text">
      <string>Roll Return Home:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="yawCheckNegative_label">
     <property name="text">
      <string>Yaw Check Negative Range:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="yawCheckPositive_label">
     <property name="text">
      <string>Yaw Check Positive Range:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="yawReturnHome_label">
     <property name="text">
      <string>Yaw Return Home:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QProgressBar" name="pitchCheckNegativeProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QProgressBar" name="pitchReturnHomeProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QProgressBar" name="yawCheckPositiveProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QProgressBar" name="yawReturnHomeProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="testStatus_label">
     <property name="text">
      <string>Test Status:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
